blog/01-18-2023-pros-and-cons-of-celo-api/pros-and-cons-of-celo-api.md

```diff
@@ -0,0 +1,129 @@
+---
+title: Pros and cons of using Celo API to create Defi applications
+description: A foray in to building on Celo using Celo's API
```

Comment:
hey @maxzysparks, the description of this tutorial diverts from the main concept, kindly change it

Reply:
okay